pallets/foreign-investments/src/lib.rs

```diff
@@ -154,20 +249,8 @@ pub mod pallet {
 			+ TypeInfo
 			+ MaxEncodedLen;
 
-		/// The default sell rate for token swaps which will be applied to all
-		/// swaps created/updated through Foreign Investments.
-		///
-		/// Example: Say this rate is set to 3/2, then the incoming currency
-		/// should never cost more than 1.5 of the outgoing currency.
-		///
-		/// NOTE: Can be removed once we implement a
-		/// more sophisticated swap price discovery. For now, this should be set
-		/// to one.
-		#[pallet::constant]
-		type DefaultTokenSellRatio: Get<Self::BalanceRatio>;
```

Comment:
BTW, I've removed this thinking that later we'll use a market ratio, and for now, all swaps are 1:1. Just for the sake of simplifying things now.